### ISSUE TITLE
Fixing a bug in portfolio optimization code

### ISSUE THREAD

**Issue by FinJun** [PredOptwithSoftConstraint/PredOptwithSoftConstraint#1]
To run the portfolio optimization experiment, I used your script. However, there was an error. It seems that the code in line 214 of main.py needs to be modified as follows.
`f_time.write('Random seed, {}, forward time, {}, inference time, {}, qp time, {}, backward_time, {}\n'.format(str(SEED), total_forward_time, total_inference_time, total_qp_time, total_backward_time))
`

**Comment by KaiYan289** [comment]
Yes you are right; thanks a lot for pointing out. This is an anonymous account though, and unfortunately I cannot retrieve the password currently. I will make changes accordingly when I retrieved the account.
